docs/data_model.md

```diff
@@ -0,0 +1,405 @@
+[< Back to Navigation](../README.md)
+
+# Data model
+
+> This document will undoubtedly change and grow as we develop our understanding of the services that this application encapsulates.
+
+This document gives an overview of our models for storing NPQ data.
+
+The aim is:
+
+- not to introduce anything unnecessary to NPQ;
+- structures the data so it's easy to understand;
+- the queries that power the API should be simple and fast;
+
+
+### Entity Relationship Diagram (ERD)
+
+This diagram represents our current understanding of the data models that will exist within this application.
```

Comment:
these models already exist, maybe we can change this description to say this is the current data model